src/circuits/query.ts

```diff
@@ -21,59 +23,65 @@ const serializationValueDataSlotBType = 'serialization:ValueDataSlotB';
 
 // Query is a query to circuit
 export interface Query {
-  allowedIssuers: string[];
+  allowedIssuers: string;
   req: Map<string, unknown>;
-  schema: Schema;
-  claimId: string;
+  context: string;
+  type: string;
+  claimID?: string;
 }
 
 // ClaimOutputs fields that are used in proof generation
 export interface ClaimOutputs {
-  issuerId: string;
-  schemaHash: bigint;
-  slotIndex: number;
+  issuerId: Id;
+  schemaHash: SchemaHash;
+  slotIndex?: number;
   operator: number;
   value: bigint[];
+  timestamp: number;
+  merklized: number;
+  claimPathKey?: bigint;
+  claimPathNotExists?: number;
+  valueArraySize: number;
 }
 
 export async function checkQueryRequest(
   query: Query,
   outputs: ClaimOutputs,
   schemaLoader: ISchemaLoader,
 ): Promise<void> {
-  const issuerAllowed = query.allowedIssuers.some(
-    (issuer) => issuer === '*' || issuer === outputs.issuerId,
-  );
-  if (!issuerAllowed) {
+  if (
+    query.allowedIssuers !== '*' &&
```

Comment:
Is  allowed issuers not an array anymore?

Reply:
it's an array.

Reply:
this is not right, it's array.
in e2e we also must accept array.

Reply:
Fixed